pkg/trebuchet/testdata/service_create.json

```diff
@@ -0,0 +1,11 @@
+{
+	"apiVersion": "creator.voyager.atl-paas.net/v1",
```

Comment:
This YAML is specific to Creator, you should have some example of Release instead (if you plan to test with it)